src/utils/transition-manager.js

```diff
@@ -132,7 +134,12 @@ export default class TransitionManager {
   }
 
   _isTransitionEnabled(props: ViewportProps): boolean {
-    return props.transitionDuration > 0 && Boolean(props.transitionInterpolator);
+    const {transitionDuration} = props;
+    return (
+      (transitionDuration > 0 ||
+        (typeof transitionDuration === 'string' && transitionDuration.toLowerCase() === 'auto')) &&
```

Comment:
To simplify things, just require the string to be strictly `'auto'`?

Reply:
Added a note under `Remarks` in `viewport-transitions.md`, to clarify `auto` works only with  `FlytoInterpolator`.